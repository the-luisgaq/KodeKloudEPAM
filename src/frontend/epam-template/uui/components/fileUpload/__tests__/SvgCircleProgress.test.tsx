import React from 'react';
import { SvgCircleProgress } from '../SvgCircleProgress';
import { render } from '@epam/uui-test-utils';

describe('SvgCircleProgress', () => {
    it('should be rendered correctly', () => {
        const { asFragment } = render(<SvgCircleProgress progress={ 35 } size={ 30 } />);
        expect(asFragment()).toMatchSnapshot();
    });
});
