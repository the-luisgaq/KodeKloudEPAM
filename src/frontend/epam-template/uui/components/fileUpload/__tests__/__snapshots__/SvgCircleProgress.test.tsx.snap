// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SvgCircleProgress should be rendered correctly 1`] = `
<DocumentFragment>
  <svg
    class="root"
    height="30"
    width="30"
  >
    <circle
      cx="15"
      cy="15"
      fill="transparent"
      r="6"
      stroke-dasharray="37.69911184307752"
      stroke-dashoffset="24.504422698000386"
      stroke-width="12"
    />
    <circle
      cx="15"
      cy="15"
      fill="transparent"
      r="14"
      stroke-width="1"
    />
  </svg>
</DocumentFragment>
`;
